src/mixins/filing-template-mixin.ts

```diff
@@ -256,16 +256,18 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           email: this.getBusinessContact.email,
           phone: this.getBusinessContact.phone,
           extension: this.getBusinessContact.extension
-        },
-        courtOrder: {
-          fileNumber: this.getFileNumber
         }
       }
     }
 
-    // Save this boolean to drafts only
-    // It is a state required for UX when resuming drafts
-    if (isDraft) filing.alteration.courtOrder.hasPlanOfArrangement = this.getHasPlanOfArrangement
+    // Apply Court Order ONLY when it is required and applied
+    if (this.getHasPlanOfArrangement || this.getFileNumber) {
+      filing.alteration.courtOrder = {
+        fileNumber: this.getFileNumber,
```

Comment:
The "if" condition above means File Number could be falsy. Is that OK here?

Reply:
Indeed, it is defined initially as falsy(`""`). Which is ok. 
Even if it is redefined here as `""`, we optionally set it when returning from DRAFT to avoid. 
Also to comment on that, i don't believe fallback handling is required there because we `set` it if it exists or don't if it doesn't. No harm